packages/flutter/test/widgets/routes_test.dart

```diff
@@ -105,6 +105,7 @@ Future<void> runNavigatorTest(
 ) async {
   expect(host, isNotNull);
   test();
+  await tester.pump();
```

Comment:
Would you like to use the changes here https://github.com/flutter/flutter/pull/68202/files#diff-694c7fe05eeca16e2ac5a7051e747cf5ff00a4108a64053b934aa38b61b0e072R105 so that it's clearer when each lines are called?

Reply:
Done.